Pipfile

```diff
@@ -15,10 +15,11 @@ pytest = "==3.10.0"
 requests-iap = "==0.2.0"
 python-helpscout-v2 = "==1.0.1"
 algoliasearch = ">=2.0,<3.0"
+typing_extensions = ">3.0"
```

Comment:
@damageboy According to [this](https://pypi.org/project/typing-extensions/)

> The typing module was added to the standard library in Python 3.5, but many new features have been added to the module since then. This means users of Python 3.5 - 3.6 who are unable to upgrade will not be able to take advantage of new types

Is this still needed for Python 3.8?